src/Passes/Source/RuleTransformationPass/Configuration.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class RuleTransformationPassConfiguration
```

Comment:
At some point we will need more descriptive doc comments for this class as well.

Reply:
Please see #1147